src/vue/forms/DepositForm.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="deposit">
+    <template v-if="isLoaded">
+      <template v-if="form.assetCode">
+        <div class="app__form-row deposit__form-row">
+          <p class="deposit__help">
+            {{ 'deposit-form.how-to' | globalize }}
+          </p>
+        </div>
+        <form>
+          <div class="app__form-row deposit__form-row">
+            <div class="app__form-field">
+              <select-field
+                :values="assetCodes"
+                v-model="form.assetCode"
+                :label="'withdrawal-form.asset' | globalize"
+                :disabled="formMixin.isDisabled"
+              />
+            </div>
+          </div>
+          <template v-for="assetCode in assetCodes">
+            <address-viewer
+              @data-loaded="enableForm()"
```

Comment:
* not clear why `address-viewer` loads something
* not clear what `data` is loaded

let's rename the component to `address-loader` and make it emit `ready` event.